includes/platform-checkout/class-platform-checkout-utilities.php

```diff
@@ -122,10 +122,16 @@ public function get_persisted_available_countries() {
 	 * @return array
 	 */
 	public function get_woopay_available_countries() {
-		$last_check = get_option( self::AVAILABLE_COUNTRIES_LAST_CHECK_KEY );
+		$next_check_option = get_option( self::AVAILABLE_COUNTRIES_NEXT_CHECK_KEY );
+		$timezone          = new \DateTimeZone( wp_timezone_string() );
+		$current_date      = new \DateTime( 'now', $timezone );
 
-		if ( $last_check && gmdate( 'Y-m-d' ) === $last_check ) {
-			return $this->get_persisted_available_countries();
+		if ( isset( $next_check_option ) ) {
+			$next_check = new \DateTime( $next_check_option, $timezone );
+
+			if ( $current_date < $next_check ) {
```

Comment:
Nitpicky and related to the above, but if we someday update the check time to once per week, the queued check will run one more time before we get to update the code to `+1 week`, right?  If we continue using `last_check` we can avoid this.